### PR TITLE
Implement `carrier apps` commands via local api request

### LINKED ISSUE #238: Implement `app` cli command through the api
The cli should talk to the api as described here: https://github.com/SUSE/carrier/pull/235#issuecomment-808060823
This story is only about the `apps` command.

This story implies the following change in commands:

- `delete` -> `app delete`
- `push` - alias to -> `app push` (keep them both)

# Acceptance Criteria

- `app delete` -> `DELETE api/v1/org/:org/applications/:name`
- `apps show` -> `GET api/v1/org/:org/applications/:name`
- `apps` -> `GET api/v1/org/:org/applications` (already implemented)

TODO: Check if it should be `applications/:name` or `application/:name` (is there a convention?)

### PR DESCRIPTION
**Pull request by jimmykarily**
Fixes #238 

Co-authored-by: Dimitris Karakasilis <DKarakasilis@suse.com>

### CODE REVIEW

**Review by rohitsakala — CHANGES_REQUESTED** [review]
This URL [0] needs to be updated.

[0] https://github.com/SUSE/carrier/blob/main/embedded-web-files/assets/js/applications.js#L8

File: acceptance/utilities_test.go [hunk 1]
```diff
@@ -51,11 +50,12 @@ func startCarrierServer(port int) (*kexec.KCommand, error) {
 
 	p := kexec.CommandString(cmd)
 
-	var b bytes.Buffer
-	p.Stdout = &b
-	p.Stderr = &b
+	p.Stdout = os.Stdout
+	p.Stderr = os.Stdout
```
[the remark this answers is not in the document]
Comment (andreas-kupries) [reply acceptance/utilities_test.go:54]: I believe @jimmykarily uses stdout for both intentionally

**Comment by andreas-kupries** [comment]
Will fix the urls in the javascript.

File: acceptance/utilities_test.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (jimmykarily) [reply acceptance/utilities_test.go:54]: I'm not sure Sterr is printed while running tests, not sure. We just need to make sure we will see the error.

[the remark this answers is not in the document]
Comment (andreas-kupries) [reply acceptance/utilities_test.go:54]: Good point. I consider this as resolved now.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**